pkg/apis/certmanager/v1/types_issuer.go

```diff
@@ -284,10 +284,14 @@ type VaultKubernetesAuth struct {
 	// A required field containing the Vault Role to assume. A Role binds a
 	// Kubernetes ServiceAccount with a set of Vault policies.
 	Role string `json:"role"`
+
+	// List of audiences to include as an audience claim in the token created
+	// +optional
```

Comment:
suggestion: This comment needs updating to match the internal/apis/certmanager version I think - that'll cause the CRDs to be updated too.